flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/SqlCommandParser.java

```diff
@@ -123,6 +123,10 @@ private SqlCommandParser() {
 			"(INSERT\\s+INTO.*)",
 			SINGLE_OPERAND),
 
+		CREATE_TABLE(
+			"(CREATE\\s+TABLE\\s+.*)",
```

Comment:
Seems this can't support `CREATE TABLE (xxxxx) WITH (xxxxx)`?

Reply:
The create table with syntax should also works fine.  because we only use this prefix to identify its whether a create table command or not,   if sure we still need to pass it to the parser & planner to get it's detailed execution.